### PR TITLE
[wip] Simple admin tool for getting signups

### PR DESCRIPTION
**Pull request by coiax**
Admins trigger a "pray for fun request", everyone gets an action button
they can trigger if they want to opt in to the fun lottery.

After a delay to signup, X number of people are selected, and the admins
are messaged with their mobs so they can PM or whisper them as
appropriate.

No action is taken on being selected other than the admins being
messaged, so the admins can feel free to request 3 people, but only pick
~~their metabuddy~~ the most suitable candidate.

Will probably make the action button a clown.

Reasons: Well, it means that everyone gets a chance for fun, rather than
the first person who can come up with the lowest common denominator of
12 and 18.

### CODE REVIEW

**Comment by kevinz000** [comment]
I'm still going to use that

**Comment by kevinz000** [comment]
Like no offense but I like giving out puzzles occasionally this isn't a bad idea but I personally like the manual way

**Comment by Niknakflak** [comment]
Rip nostalgia

**Comment by Supermichael777** [comment]
3

**Comment by lzimann** [comment]
@kevinz000 i mean it won't force any admin to use this feature so not a big deal

**Comment by PKPenguin321** [comment]
The nice thing about asking for prayers is that if you wanna toss up traitor status or something you can sift through the prayers ignoring ineligible roles and pick one that would work (so ignore ghosts and the HoS but pick a random botanist). This might allow for that if you enter a higher value than you need, but whatever.

**Comment by Armhulen** [comment]
I have 4x4^2 as a macro, checkmate kevinz

**Comment by kevinz000** [comment]
cue me asking for 4*4^2 and laughing when people say 256 <3

File: code/modules/admin/pray_for_fun.dm [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+/datum/action/pray_for_fun
+	name = "Pray for Fun!"
+
+/datum/action/pray_for_fun/IsAvailable()
+	return TRUE
+
+/datum/action/pray_for_fun/Trigger()
+	if(!..())
+		return
+	target += owner
```
Comment (RemieRichards) [inline code/modules/admin/pray_for_fun.dm]: This is disgusting.

**Comment by AnturK** [comment]
At a minimum you should be able to specify ghosts/alive/everyone filter.

**Comment by KorPhaeron** [comment]
This would be a useful button, please let me know when this compiles/is ready

**Comment by coiax** [comment]
Stop trying to reduce the number of open PRs by closing in progress PRs

**Comment by KorPhaeron** [comment]
We have a limited amount of time and effort to dedicate to PR review, the PR compiling at all should be the bare minimum before you ask us for some of that time

**Pull request closed without merging**